src/arrayMethodSort.js

```diff
@@ -5,7 +5,22 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (
+          compareFunction
+            ? compareFunction(this[i], this[j]) > 0
+            : String(this[i]) > String(this[j])
```

Comment:
It's better to create a default comparing function outside applyCustomSort function and set it as a default value

Example: 
```
const compareAsString = (a, b) => {
...
}

...

[].__proto__.sort2 = function(compareFunction = compareAsString) {
...
```